.php_cs

```diff
@@ -12,6 +12,7 @@ EOF;
 
 $finder = Symfony\CS\Finder\DefaultFinder::create()
     ->in(array(__DIR__.'/src', __DIR__.'/tests'))
+    ->exclude(array(__DIR__.'/tests/fixtures/'))
```

Comment:
That's strange, but despite this line, `php-cs-fixer` still want me to update the `tests/fixtures/Command/DiffTest/AcmeProject/src/App/banana.php` file to add the header.

Any hint about what I am missing ? 😕